content/syllabuses/employed-java-post-skills-test.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Employed Java post skills test
+---
+
+- {{% contentlink path="topics/clean-code/java" %}}
+- {{% contentlink path="topics/bootcamp/git-for-bootcamp" %}} TODO:  recreate with examples & video on how to submit - drag and drop files
+- {{% contentlink path="projects/katas/level-0" flavour="java" %}}
```

Comment:
post means after... are you saying that the level 0 katas will happen after the skills test? Aren't those the ACTUAL skills test?